parse_test.go

```diff
@@ -149,6 +149,25 @@ func TestParseFile_SkipProcessingPixelDataValue(t *testing.T) {
 			}
 		})
 	})
+	t.Run("WithAllowErrorMetaElementGroupLength", func(t *testing.T) {
+		runForEveryTestFile(t, func(t *testing.T, filename string) {
+			dataset, err := dicom.ParseFile(filename, nil, dicom.AllowErrorMetaElementGroupLength())
+			if err != nil {
+				t.Errorf("Unexpected error parsing dataset: %v", dataset)
+			}
+			el, err := dataset.FindElementByTag(tag.PixelData)
+			if err != nil {
+				t.Errorf("Unexpected error when finding PixelData in Dataset: %v", err)
+			}
+			pixelData := dicom.MustGetPixelDataInfo(el.Value)
+			if pixelData.IntentionallyUnprocessed {
+				t.Errorf("Expected pixelData.IntentionallyUnprocessed=false when TestParseFile_SkipProcessingPixelDataValue option not present, got true")
+			}
+			if len(pixelData.Frames) == 0 {
+				t.Errorf("unexpected frames length when TestParseFile_WithAllowErrorMetaElementGroupLength=true. got: %v, want: >0", len(pixelData.Frames))
+			}
```

Comment:
no need to include these checks--a basic sanity check of no error is fine for this level of check, or if anything, comparing the datasets parsed with your option to the ones without your option should be identical (that would look a little like this https://github.com/suyashkumar/dicom/blob/48ba291715cc85f31c5133f81fd9d5d5dc720d2c/parse_internal_test.go#L16)